cocos/2d/renderer/static-vb-accessor.ts

```diff
@@ -51,12 +51,15 @@ const _entryPool = new Pool<IFreeEntry>(() => ({
  */
 export class StaticVBChunk {
     // public ib: Uint16Array;
+    //
+
     constructor (
         public vertexAccessor: StaticVBAccessor,
         public bufferId: number,
         public vertexOffset: number,
         public vb: Float32Array,
         indexCount: number,
+        public meshBuffer: MeshBuffer,
     ) {
         // this.ib = new Uint16Array(indexCount);
```

Comment:
Add an assert to check whether `meshBuffer` is `vertexAccessor.`getMeshBuffer(bufferId)`